### ISSUE TITLE
I can't uninstall any application

### ISSUE THREAD

**Issue by CODEX-cpp** [0x192/Universal-Android-Debloater#387]
I download the program, I start it in administrator, I select the apps I wanted to uninstall, I press the button to uninstall the selected apps, and nothing happens. I go to see on the log file and it is full of errors.
[UAD_20220921.log](https://github.com/0x192/universal-android-debloater/files/9620343/UAD_20220921.log)

**Comment by 0x192** [comment]
```
 User version can not delete system package
```

This is the first time I see this error message. It looks like TCL is now preventing users to perform a `pm uninstall` on system packages.

Try to switch to the disable mode from the settings. [It seems to work](https://android.stackexchange.com/questions/247076/cannot-remove-default-apps-on-tcl-30-xe-5g-using-adb-shell)

**Comment by CODEX-cpp** [comment]
> ```
>  User version can not delete system package
> ```
> 
> This is the first time I see this error message. It looks like TCL is now preventing users to perform a `pm uninstall` on system packages.
> 
> Try to switch to the disable mode from the settings. [It seems to work](https://android.stackexchange.com/questions/247076/cannot-remove-default-apps-on-tcl-30-xe-5g-using-adb-shell)

disabling apps is not what I wanted... also because they find the way sooner or later to reactivate themselves

**Comment by 0x192** [comment]
Yes but as you can't do `pm uninstall`, it's better to try `pm disable-user` than do nothing. And I'd really like to know if `pm disable-user` works on your phone.

> also because they find the way sooner or later to reactivate themselves

Know that even with `pm uninstall`, some "uninstalled" apps will be reactivated if they are updated through a system update (and you shouldn't delay system updates as they also contain security fixes)

**Comment by CODEX-cpp** [comment]
> pm disable-user

yes the command to disable applications works with system apps that do not uninstall
